crates/types/src/bindings.rs

```diff
@@ -162,4 +218,4 @@ impl Orders::Swap {
     pub const fn target_chain_id(&self) -> u64 {
         self.targetChainId.as_limbs()[0]
     }
-}
+}
```

Comment:
need the `Orders` impls here too

Reply:
the `sol!` block for `Orders` should be deleted, now replaced by the `RollupOrders` block